bld/CLMBuildNamelist.pm

```diff
@@ -4302,21 +4302,33 @@ sub setup_logic_fates {
               # spit-fire can't be on with FATES SP mode is active
               if ( $nl->get_value('fates_spitfire_mode') > 0 ) {
                     $log->fatal_error('fates_spitfire_mode can NOT be set to greater than 0 when use_fates_sp is true');
+              }
            }
-	 }
         }
         my $var = "use_fates_inventory_init";
         if ( defined($nl->get_value($var))  ) {
            if ( &value_is_true($nl->get_value($var)) ) {
               $var = "fates_inventory_ctrl_filename";
-	      my $fname = substr $nl->get_value($var), 1, -1;  # ignore first and last positions of string because those are quote characters
+              my $fname = substr $nl->get_value($var), 1, -1;  # ignore first and last positions of string because those are quote characters
               if ( ! defined($nl->get_value($var))  ) {
                  $log->fatal_error("$var is required when use_fates_inventory_init is set" );
               } elsif ( ! -f "$fname" ) {
                  $log->fatal_error("$fname does NOT point to a valid filename" );
               }
            }
         }
+        my $var = "use_fates_luh";
+        if ( defined($nl->get_value($var))  ) {
+           if ( &value_is_true($nl->get_value($var)) ) {
+              $var = "fluh_timeseries";
+              my $fname = substr $nl->get_value($var), 1, -1;  # ignore first and last positions of string because those are quote characters
```

Comment:
Use the remove_leading_and_trailing_quotes function to do this.

Reply:
addressed via 3bbbc53590a480a73c541d4454818102a591d255